src/components/paradox/results/personality/career/modal/employers.js

```diff
@@ -18,7 +21,8 @@ CareerModalEmployers.propTypes = {
   employers: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
-      description: PropTypes.string.isRequired
+      description: PropTypes.string.isRequired,
+      url: PropTypes.string.isRequired
```

Comment:
I think the URL shouldn't be required (the code above works without it already)

Reply:
Good catch! Ok Removed!